### DISCUSSION TITLE
How to login to it & what’s the user and password?

### DISCUSSION THREAD

**Discussion by DZ65367709** [gramps-project/gramps-web#162] [General]
As title. Thank you for your help.

**Comment by DavidMStraub** [comment]
Login to what :roll_eyes:

**Comment by DZ65367709** [comment]
As the default, once open the front end url, and there pops the login dialogue which asks for User and Password. I mean this. Is there a default one for the first login?
![Uploading 2B07A6F8-FDCC-4178-A212-89F7ED4BF93C.png…]()

**Comment by DZ65367709** [comment]
![52908EBC-0B65-43EF-A1B4-5BE376FF846C](https://user-images.githubusercontent.com/31399475/211433461-07eafe25-7142-49dc-b65f-2a627178c321.png)

**Comment by DZ65367709** [comment]
Thank you David. I think I found it out eventually. Lots of documentation after checked…
Little by little I am now starting learning more about gramps, it’s really a great job you’ve done! With appreciation and respect, I have to say. 
As I can’t deploy Gramps in a docker container or Ubuntu-docker -as there’s no GUI support- I have installed it on Windows  OS,- on a virtual machine, though it occupies more resources than on a linux server, I can use it now! 
Thanks for all you did!
![EAEA746F-49D2-4D96-86AE-0AE1E24908B8](https://user-images.githubusercontent.com/31399475/211439823-6ed08751-f1a1-45b6-8e18-1372530a6d5e.png)
